installscripts/wizard/run.py

```diff
@@ -16,6 +17,25 @@ def main():
         if len(sys.argv) > 3:
             os.environ['CODE_QUALITY'] = sys.argv[3]
 
+        if len(sys.argv) > 4:
+            input_tags = validate_tags.prepare_tags(sys.argv[4])
+            try:
+                aws_tags, aws_formatted_tags = ast.literal_eval(str(validate_tags.validate_replication_tags(input_tags)))
+                installervarsjson = sys.argv[2] + "/installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json"
```

Comment:
Please try to avoid directly updating Chef resources in the installer wizard. Wizard -> Terraform -> Chef. Each layer should only know about/touch resources for the layer directly after it. We aren't following this 100% but we should be, and there's no reason to worsen the problem if we can avoid doing so.

Do it in Terraform (e.g. `jenkins.tf`), which both a better logical spot for it and is where we are already making most of the changes to this file.

Reply:
Done